themes/tiddlywiki/vanilla/base.tid

```diff
@@ -1792,6 +1791,12 @@ html body.tc-body.tc-single-tiddler-window {
 	width: 100%;
 }
 
+.tc-controlpanel svg.tc-image-button{
+	width: 1em;
+	height: 1em;
+	vertical-align:middle;
```

Comment:
@pmario

> it's the input box, that needs a vertical-alignment.

Which input box? This is meant to vertically align the buttons in the toolbar configuration, see updated demo http://1954.tiddlyspot.com.